### ISSUE TITLE
Unable to use --exfil when specifying credentials

### ISSUE THREAD

**Issue by qgrosperrin** [Flangvik/TeamFiltration#32]
When I specify creds for the exfil module, I cannot get past the PrepareExfilCreds call. It just returns the error below. Any ideas, I have tried playing with the code but cannot resolve this... :/
```
[+] Args parsed --outpath . --exfil --all --username <username> --password <password> --debug

[!] The exfiltration modules does not use FireProx, ORIGIN IP WILL BE LOGGED, are you an adult? (Y/N)
Y
[SPRAY] NONE         25/04/2023 05:58:17 EST SOFT ERROR when spraying <username>:<password> => Invalid URI: The URI is empty.
```

**Comment by qgrosperrin** [comment]
Ok, I have pin it down to the proxy.
Each time the proxy is added for the httpClientHandler, I get a similar error. Not sure how to disable it.. I have used a config when the proxy string was just empty but that didn't work, so I have manually removed it everywhere in the solution

**Comment by Flangvik** [comment]
Thanks for opening a issue, I'll peek into this. For now you should be able to get around this by identifying the user using the spray module first, then selecting it after getting asked by '--exfil'

**Comment by Flangvik** [comment]
Issue found. When using `--debug` TeamFiltration looks for the proxy address in the config, you have not provided a config using `--config`. Normally TeamFiltration would complain about not having a config, but it was found that the exfil module should be able to run without a config. However in this cases there is missing logic to correct for this in combination with the --debug option.

**Comment by Flangvik** [comment]
Attempted fixed in [e125f79](https://github.com/Flangvik/TeamFiltration/commit/e125f79a566ab9e7c0d3f4845fac48bc4af9f58c) , closing for now
